copilot/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/protobuf v1.4.2
 	github.com/imdario/mergo v0.3.9 // indirect
-	github.com/lyft/flyteidl v0.17.32
+	github.com/lyft/flyteidl v0.17.37-0.20200727195325-5aef14f0da69
```

Comment:
once we merge flyteidl, we can just get this updated

Reply:
you should update this now? - Flyteidl is merged

Reply:
yes